src/foundry/client-esm/helpers/compendium-art.d.mts

```diff
@@ -1,3 +1,47 @@
-declare class CompendiumArt extends Map {}
+import type { CompendiumArtDescriptor, CompendiumArtInfo, CompendiumArtMapping } from "./_types.d.mts";
+
+/**
+ * A class responsible for managing package-provided art and applying it to Documents in compendium packs.
+ */
+declare class CompendiumArt extends Map<string, CompendiumArtInfo> {
+  /**
+   * The key for the package manifest flag used to store the mapping information.
+   * @defaultValue `"compendiumArtMappings"`
+   */
+  FLAG: string;
+
+  /**
+   * The key for the setting used to store the World's art preferences.
+   * @defaultValue `"compendiumArtConfiguration"`
+   */
+  SETTING: string;
```

Comment:
SETTING is readonly since it's used to get something in the Settings singleton. Strictly speaking, FLAG shouldn't break everything if changed, but I think it's reasonable to put it as readonly.